### PR TITLE
Optimize login registration style

### PR DESCRIPTION
**Pull request by zli213**

### CODE REVIEW

**Review by kesesek — APPROVED** [review]

File: public/images/right_img.png
Comment (jlmjkfd) [inline public/images/right_img.png:1]: Does the background needs to be transparent?

**Review by jlmjkfd — APPROVED** [review]

**Comment by ryanwooa** [comment]
image
- Position of login card is different in both pages. How about make them same position.
- Does the page require a title, otherwise the user will not know if it is currently a login page or a registration page

File: public/images/right_img.png
Comment (zli213) [reply public/images/right_img.png:1]: Deleting unnecessary photo.

**Comment by zli213** [comment]
> image
> 
> * Position of login card is different in both pages. How about make them same position.
> * Does the page require a title, otherwise the user will not know if it is currently a login page or a registration page

Fixed.

**Review by ryanwooa — APPROVED** [review]

**Pull request merged**